ci-operator/step-registry/openshift/e2e/azure/sdn/techpreview/serial/openshift-e2e-azure-sdn-techpreview-serial-workflow.yaml

```diff
@@ -7,6 +7,8 @@ workflow:
     - ref: sdn-conf
     - chain: ipi-install
     - ref: ipi-azure-rbac
+    - ref: platformoperator-valid-cert-manager
+    - chain: ipi-azure-pre
```

Comment:
@jianzhangbjz it is also duplicated and cause job failure.
maybe you should also remove it.

Reply:
That's really strange! Anyway, I removed it.